packages/web/components/Quest/QuestForm.tsx

```diff
@@ -355,6 +373,47 @@ export const QuestForm: React.FC<Props> = ({
           </FlexContainer>
         </Field>
 
+        <Field label="Quest Image" error={errors.image}>
+          <Input
+            {...register('image', {
+              required: true,
+            })}
+            type={'file'}
+            paddingTop={1}
+            accept="image/*"
+            onChange={(e) => showImagePreview(e)}
+          />
+          <Center
+            boxSize="sm"
+            rounded={'md'}
+            border={'dashed'}
+            borderWidth={6}
+            borderColor={'whiteAlpha.500'}
+            marginTop={2}
+            height={'xs'}
+            width={'full'}
+            padding={2}
+            overflow="clip"
+            backgroundColor={'blackAlpha.600'}
+            backdropFilter={'auto'}
+            backdropBlur={'sm'}
+          >
+            {previewImg ? (
+              <Image
+                transition={'ease-in'}
```

Comment:
Is this the right format for the [`transition` property](https://drafts.csswg.org/css-transitions/#transition-shorthand-property)?

Should it be:
```suggestion
                transition="all 0.6s ease-in"
```